README.md

```diff
@@ -19,38 +19,39 @@
 
 ## Usage
 
-### Workflow
+### Scan CI Pipeline (w/ Trivy Config)
```

Comment:
It would be nice to keep the 2 pipeline samples, before the deprecation of the other workflow.

Reply:
@simar7 0.30 It was released!

Reply:
> It would be nice to keep the 2 pipeline samples, before the deprecation of the other workflow.

Yeah good point. I'll add that back in.